### DISCUSSION TITLE
Tokens are not accepted

### DISCUSSION THREAD

**Discussion by douwevt** [teslamate-org/teslamate#2196] [General]
When trying to log on to Teslamate (existing installation that did work before) I get an error saying: "Error: Tokens are invalid". I used the tooling tesla_auth on Windows, the IOS app "Auth for Tesla" and the token generator on tesla-info.com but they all end up in not being able to log on. Anyone any idea on how to solve this? Thanks, Douwe

**Comment by manuelkamp** [comment]
The easiest way, without knowing the issue, will be backing up your DB and deleting all docker volumes. start again, and import the database backup. If you connect to the dashboard, it will ask for tokens and you should be able to log in again. it takes 20 minutes if you are slow.

**Reply by douwevt** [reply]
Thank you ManuelKamp,
It worked indeed, I did not even have to restore from backup. All in all, less than 15 minutes, and I am not a Docker guru. 
Regards, Douwe

**Reply by miles267** [reply]
Tried to do this just now @manuelkamp.  Is there a guide on how to do it?  When I did it, it still didn't accept my Tesla v3 tokens although I'm running the latest Teslamate 1.25.0

**Reply by douwevt** [reply]
I think I found the problem, my Teslamate server (Raspberry Pi) uses a VPN to connect to the internet. Once this is started, the connection between Teslamate and the Tesla is dropped. I did not find a way yet to circumvent that issue, except not starting the VPN service.

**Comment by henryhm0** [comment]
Had the same issue today on Synology. I restarted synology DS and the problem was sorted.

**Comment by stefaanbolle** [comment]
I can't get around it. A backup ad restore on a QNAP in docker leaves tokens invalid

**Comment by firefountain** [comment]
VPN.. somehow. If you are using a VPN it never connects..

**Reply by cwanja** [reply]
Tesla blocks accessing their owner side via VPNs. Sure there are new VPNs that they have not blocked.
